### PR TITLE
Add page title and fixup index

### PR DESCRIPTION
**Pull request by kareiva**
* Add page title so that different H1's do not break UX
* Replace main link to direct URL for log upload (TODO: replace it with blue <button>)

### CODE REVIEW

File: pages/index.md [hunk 1]
```diff
@@ -7,7 +7,11 @@ titleTemplate: false
 
 Welcome to the NRAU Baltic contest 2024 on the 14th of January.
 
-This year's host is SSA - Sweden. Log upload and results can be found here: https://logs.nraubaltic.eu/.
+This year's host is SSA - Sweden. Log upload and results can be found on the website https://<span>logs.nraubaltic.eu/</span>.
```
Comment (domnantas) [inline pages/index.md:10]: Why did you make this url non-clickable?

**Comment by kareiva** [comment]
To guide people towards log upload and avoid questions "where do I upload
logs".

**Comment by kareiva** [comment]
Now with the proposed font it looks good to me :)

File: .vitepress/theme/Layout.vue [hunk 2]
```diff
@@ -22,12 +22,16 @@ const navigationItems = {
...
-			<a href="/" class="mr-4">
+			<a
+				href="/"
+				class="mr-4 flex items-center gap-4 font-semibold text-lg text-gray-700 dark:text-gray-200"
+			>
 				<img
 					class="w-10 h-[78px]"
 					src="/logo.svg"
-					alt="NRAU-Baltic contest logo"
+					alt="NRAU Baltic contest logo"
```
Comment (kareiva) [inline .vitepress/theme/Layout.vue]: Contest name is NRAU-Baltic

**Pull request merged**